api/graphql/schema.graphql

```diff
@@ -2603,7 +2603,9 @@ input WorkExperienceInput {
   cafForce: CafForce @rename(attribute: "caf_force")
   cafRank: CafRank @rename(attribute: "caf_rank")
   classificationId: String @rename(attribute: "classification_id")
-  departmentId: String @rename(attribute: "department_id")
+  departmentId: DepartmentBelongsTo
+    @pluck(key: "connect")
+    @rename(attribute: "department_id")
```

Comment:
For some reason this didn't work otherwise, don't know why this needed unique directives appended